index.js

```diff
@@ -26,17 +27,30 @@ class Task {
         item.append(deleteBtn);
         taskInner.append(item);
 
+        deleteBtn.addEventListener('click', this.deleteTask)
+
         addInput.value = '';
     }
+
+    deleteTask(e) {
+        const res = tasks.filter(item => {
+            return item.id !== this.id;
+        })
+        tasks = res;
+        e.target.parentNode.remove();
+    }
 }
 
 const addTask = () => {
     const value = addInput.value;
     if (value !== '') {
-        const newTask = new Task(value);
+        let id = Number(new Date);
```

Comment:
If you wanna to get timestamp you can use Date.now(). And lets make this variable as const